README.md

```diff
@@ -138,6 +138,10 @@ Of course not. IDLE is an awful mess that you should stay far away from.
 ### Can I play tetris with Porcupine?
 Of course, just install the tetris plugin. See [more_plugins](more_plugins/).
 
+### Is Porcupine a toy project or is it meant to be a serious editor?
+No. Porcupine is indeed meant to be a serious editor, in fact you might regret even touching it.
```

Comment:
A "yes/no" answer to a "foo or bar" question doesn't make much sense to me. How about this?

```suggestion
Porcupine is meant to be a serious editor, in fact you might regret even touching it.
```

Reply:
I approve :)